source/MRMesh/MRMeshBoolean.cpp

```diff
@@ -385,6 +391,25 @@ BooleanResultPoints getBooleanPoints( const Mesh& meshA, const Mesh& meshB, Bool
         collFacesA = fillContourLeft( meshA.topology, collBordersA );
         result.meshAVerts = getInnerVerts( meshA.topology, collFacesA );
 
+        std::function<void ( EdgeId )> removeDeadEndsA;
+        removeDeadEndsA = [&] ( EdgeId e0 )
```

Comment:
Mb make this function take topoligy and needed bitset as args, not to duplicate it